src/main/webapp/feed.html

```diff
@@ -10,39 +10,15 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="getNextPotentialMatch()">
     <div class="container-fluid">
       <div data-include="nav.html"></div>
       <div class="content">
         <h1>Feed</h1>
         <div id="feed" class="carousel slide" data-ride="carousel">
-          <ul class="carousel-indicators">
-            <li data-target="#feed" data-slide-to="0" class="active"></li>
-            <li data-target="#feed" data-slide-to="1"></li>
-            <li data-target="#feed" data-slide-to="2"></li>
+          <ul class="carousel-indicators" id="carousel-indicators">
```

Comment:
class is for bootstrap stuff, id is because I was originally doing document.getElementsByClassName on these but that wasn't working for appenChild() and when I tried document.getElementById it did work.